src/components/icons/magnifier_icon.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const MagnifierIcon: React.FC<React.SVGAttributes<SVGElement>> = (props) => (
```

Comment:
Could you explain why we have two magnifier icons?
I've put these two icons in the browser to see how they look like. The difference between them is that the bigger one has more details. Just wondering the reason why we need two magnifier icons...

Reply:
Displaying the more detailed icon at small sizes makes it look ugly so the designers have come up with icons that should be used up to 24x24 pixels and 25x25 and up.